arduino/src/arduino.cpp

```diff
@@ -0,0 +1,29 @@
+#include "arduino.h"
+
+Arduino::Arduino(){
+    Arduino::portName = "/dev/ttyUSB0";
+    Arduino::buadRate =9600; 
+    Arduino::timeout = 10000;
+
+}
+
+Arduino::Arduino(std::string fileToWrite,std::string portName, unsigned int buadRate, unsigned int timeout){
+    Arduino::fileToWrite.open(fileToWrite.c_str());
+
+}
+int Arduino::checkConnection(){
+    char helper = Arduino::serial.openDevice(portName.c_str(),buadRate);
```

Comment:
the use of `ClassName::Something` is for static but this is not to be use with anything that is not static for not static you need to write `Something` or `this->Something`.